appengine_config.py

```diff
@@ -71,12 +71,13 @@ def save(self):
 # Root path of the app.
 ROOT_PATH = os.path.dirname(__file__)
 
-_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
-pil_path = os.path.join(_PARENT_DIR, 'oppia_tools', 'PIL-1.1.7')
+if feconf.DEV_MODE:
```

Comment:
Actually, maybe you had better change this to "if oppia_tools dir exists" instead.

That's because when the server is run locally in --prod_env mode, PIL doesn't exist. All e2e tests are currently failing.